README.md

```diff
@@ -89,7 +95,6 @@ After it finishes, you can find your images in the `output` directory on your ho
 
 
 ### Known issues
```

Comment:
Shoudn't we comment [this issue](https://tracker.mender.io/browse/MEN-2399?focusedCommentId=96474&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-96474)?

Reply:
Short note also added.

Reply:
I can't see it

Reply:
Please check now.